lua/lspconfig/server_configurations/lua_ls.lua

```diff
@@ -1,13 +1,16 @@
 local util = require 'lspconfig.util'
 
 local root_files = {
+  '.busted',
+  '.git',
+  '.luacheckrc',
   '.luarc.json',
   '.luarc.jsonc',
-  '.luacheckrc',
   '.stylua.toml',
-  'stylua.toml',
+  'config.ld', -- ldoc
```

Comment:
this not make sense .

Reply:
`config.ld` is the project-level config file for ldoc